ci-operator/step-registry/cucushift/installer/check/ibmcloud/kms-key/cucushift-installer-check-ibmcloud-kms-key-commands.sh

```diff
@@ -24,31 +24,94 @@ critical_check_result=0
 
 key_file="${SHARED_DIR}/ibmcloud_key.json"
 
-id=$(jq -r .id ${key_file})
-if [[ -z $id ]]; then
-    echo "[ERROR] fail to find kp instance id !!"
-    exit 1
+echo "ControlPlane EncryptionKey: ${IBMCLOUD_CONTROL_PLANE_ENCRYPTION_KEY}"
+echo "Compute EncryptionKey: ${IBMCLOUD_COMPUTE_ENCRYPTION_KEY}"
+echo "DefaultMachinePlatform EncryptionKey: ${IBMCLOUD_DEFAULT_MACHINE_ENCRYPTION_KEY}"
+
+cat ${key_file}
+
+id_m=""
+id_w=""
+id_d=""
+
+if [[ "${IBMCLOUD_DEFAULT_MACHINE_ENCRYPTION_KEY}" == "true" ]]; then
+    id_w=$(jq -r .default.id ${key_file})
```

Comment:
here should be `id_d`, right?

Reply:
IBMCLOUD_DEFAULT_MACHINE_ENCRYPTION_KEY will be used worker, master and default
IBMCLOUD_CONTROL_PLANE_ENCRYPTION_KEY can override key of master 
IBMCLOUD_COMPUTE_ENCRYPTION_KEY can override key of worker

Reply:
From your var naming, `id_d` is to save `IBMCLOUD_DEFAULT_MACHINE_ENCRYPTION_KEY`, right?

Reply:
id_d save the key used in data volume, if provided by the key of DEFAULT_MACHINE